src/index.js

```diff
@@ -0,0 +1,190 @@
+const defaultCity = 'Seattle';
+const state = { temp: '', currentCity: defaultCity };
+
+const currentTemp = document.getElementById('current-temp');
+const upButton = document.getElementById('up-temp');
+const downButton = document.getElementById('down-temp');
+
+const currentLandscape = document.getElementById('landscape-weather-visual');
+const currentSky = document.getElementById('sky-weather-visual');
+const skySelector = document.getElementById('sky-select');
+
+const currentCityDisplay = document.getElementById('city');
+const cityInput = document.getElementById('city-input');
+const resetCityButton = document.getElementById('city-reset');
+
+const realTimeButton = document.getElementById('real-time-button');
+
+const dateTime = document.getElementById('date-time');
+const currentDate = new Date().toUTCString();
+dateTime.textContent = currentDate.slice(0, 3) + ' ' + currentDate.slice(5, 16);
```

Comment:
Ooo, displaying the date and time is actually one of my favorite website flourishes! I'm so delighted!!